src/Symfony/Component/PropertyAccess/Tests/Fixtures/TestSingularAndPluralProps.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Tests\Fixtures;
+
+/**
+ * Notice we don't have getter/setter for emails
+ * because we count on adder/remover.
+ */
+class TestSingularAndPluralProps
+{
+    /** @var string|null */
+    private $email;
+
+    /** @var array */
+    private $emails = array();
+
+    /**
+     * @return string|null
+     */
+    public function getEmail(): ?string
```

Comment:
thanks for the PR.
If the issue exists on 3.4, can you please rebase+retarget your PR for branch 3.4?
This will require writing it using PHP5.5 compatible code (that's why I'm commenting on this line :) )

Reply:
maybe close this PR and submit a new one would be the easiest?

Reply:
I sent my changes in another PR https://github.com/symfony/symfony/pull/28966